Kaetram/Kaetram-Open
# add: bracelets, pendants and rings

### Motivation

Add bracelets, pendants and rings

**How to test (feature)**

-Observe that new pictures (PNG) of bracelets, pendants, and rings are displayed correctly.

**How to test (potential regressions)**

-Navigate to various other pages and sections of the application.
-Verify that the changes made to the items.json and sprites.json files do not cause any unintended side effects, such as broken links or display issues.

### Changes

Please describe any changes to the code you have made.

-Added new pictures (PNG) of bracelets, pendants, and rings.
-Made modifications to the items.json file.
-Updated the sprites.json file to reflect the changes
